tests/__init__.py

```diff
@@ -0,0 +1 @@
+"""Required to ignore mypy errors in the tests folder."""
```

Comment:
Linters and mypy. See comment with e267eddc59ebc07608c3cf36a2f5cb84dde4bc38

Reply:
But e267eddc59ebc07608c3cf36a2f5cb84dde4bc38 can be kicked out, I have not found a better solution to let mypy ignore the tests folder

Reply:
i would prefer a better solution

Reply:
mypy closed the issue, afaics they think this should be the way to do it.
Alternatively, we can add type hints to tests.

Reply:
I see. Would it be possible to remove the `__init__.py`'s in the subdirectories?

Reply:
I tried that as well (so only `__init__.py` in the `tests` dir) because I also hate this 'solution', but no success. Weirdly, also
```ini
files =
    expertsystem/**/*.py,
    expertsystem/**/*.pyi,
```
in the `tox.ini` config doesn't work.

Reply:
> Alternatively, we can add type hints to tests.

But this is a serious suggestion: they may help the readability of the tests. Although... the main reason for starting to use type hints was the API.

Reply:
Hmm nah, that's a waste of time.

Reply:
I agree.
So to conclude, it seems that mypy is tuned to work with modules, which is why it requires init files (even if empty dummies). See for instance also
https://stackoverflow.com/a/60998686

Reply:
Ok so lets use the regular package for tests as a workaround for now until this issue (https://github.com/python/mypy/issues/5759) is fixed.

Note: the problem that  `namespace_packages = True` does not work is related to pre-commit calling a file explicitly.